apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.Snippets/MetadataClientSnippets.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Compute.v1.Data;
+using Google.Cloud.Metadata.V1;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace Google.Cloud.Metadata.V1.Snippets
+{
+    [Collection(nameof(MetadataSnippetFixture))]
+    public class MetadataClientSnippets
+    {
```

Comment:
I'd normally expect a constructor taking a MetadataSnippetFixture here, setting in as a field.
I'm slightly curious that this works, but I believe you :)